### DISCUSSION TITLE
DSEP Working Group | Call 8 | 06 April 2022

### DISCUSSION THREAD

**Discussion by sujitrs** [beckn/DSEP-Specification#27] [General]
![image](https://user-images.githubusercontent.com/23429265/161942516-1d6866fb-e66b-42b7-9cae-35ae59bc9f60.png)

**Comment by sujitrs** [comment]
Call Deck, Attendees, Important Bookmarks, Categorised and Consolidated List of Use Cases for Implementation. Call Recording shall be added subsequently.
[DSEP Working Group - Call 8 - April 06, 2022.pdf](https://github.com/beckn/DSEP-Specification/files/8427143/DSEP.Working.Group.-.Call.8.-.April.06.2022.pdf)

**Comment by SaketSinha** [comment]
It appears the call recoding link is not part of the PDF meeting notes.
@sujitrs - Can you please share the same.

Thanks

**Reply by SaketSinha** [reply]
Hi @sujitrs - Please help me with the recording link.

**Reply by sujitrs** [reply]
Sure

**Reply by sujitrs** [reply]
@core-wg-admin please share...

**Reply by SaketSinha** [reply]
Gentle reminder

**Reply by ravi-prakash-v** [reply]
https://drive.google.com/file/d/1qO-a_regqXz8QzhMj45ia1p0ANSMiS2F/view?usp=sharing
